packages/web-react/src/components/ModalComposed/ModalBody.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
I think this implementation is the same as the `Modal/ModalBody.tsx` component. I suggest using the older one instead.

Reply:
After the discussion with Adam, we should not give a choice of element type for Body. That's why I'd rather use the new version. At the same time, I extended the types with native DIV properties. No need to use TransferProps.

Reply:
Ok.